Generation/options/particleGun.py

```diff
@@ -0,0 +1,53 @@
+from Gaudi.Configuration import *
+from Configurables import ApplicationMgr, THistSvc, Gaudi__ParticlePropertySvc
+from Configurables import HepMCDumper, ParticleGunAlg, MomentumRangeParticleGun, HepMCHistograms, FlatSmearVertex
+
+dumper = HepMCDumper("Dumper")
+dumper.DataInputs.hepmc.Path="hepmc"
+
+
+# A particle table file needs to be present in the directory 
+# from which Gaudi is run. If you run from the top-level directory,
+# the following option can be added to the service to use the file in this directory.
+#ParticlePropertiesFile='Generation/options/ParticleTable.txt'
+
+particlePropertySvc = Gaudi__ParticlePropertySvc("ParticlePropertySvc", ParticlePropertiesFile='/afs/cern.ch/lhcb/software/releases/GAUDI/GAUDI_v26r4/GaudiExamples/tests/data/ParticleTable.txt')
```

Comment:
Right, I wanted to ask you about that. CTest was failing even with WORKING_DIRECTORY set the path of the file. I'll check again.